core/src/main/java/ai/timefold/solver/core/config/solver/SolverConfig.java

```diff
@@ -60,6 +61,7 @@
  */
 @XmlRootElement(name = SolverConfig.XML_ELEMENT_NAME)
 @XmlType(name = SolverConfig.XML_TYPE_NAME, propOrder = {
+        "enablePreviewFeatureList",
```

Comment:
I'd drop the `list` - or do we have a precedent for that anywhere else in the config?

Reply:
We have `acceptorTypeList`, which is also a `List`.

Reply:
In that case, I think it should be `enabledPreviewFeatureList`.